dev/io.openliberty.microprofile.telemetry.1.0.internal.jaeger_fat/fat/src/io/openliberty/microprofile/telemetry/internal/utils/otelCollector/OtelCollectorContainer.java

```diff
@@ -74,4 +74,15 @@ public int getOtlpGrpcPort() {
     public String getOtlpGrpcUrl() {
         return "http://" + getHost() + ":" + getOtlpGrpcPort();
     }
+
+        /**
+     * Get the URL to use to send OLTP spans via gRPC over tls
+     * <p>
+     * Only valid when the container is started
+     *
+     * @return the OLTP gRPC URL
+     */
+    public String getSecureOtlpGrpcUrl() {
+        return "https://" + getHost() + ":" + getOtlpGrpcPort();
+    }
```

Comment:
Is this bit used anywhere?

Reply:
I don't think its used yet, but it will be if we write a similar test for Grpc.